apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/ExplanationOfBenefitE2E.java

```diff
@@ -65,7 +76,7 @@ public void setupTest() {
    */
   @Test
   public void testReadEobCarrierIdExpectClmIdInResponse() {
-    verifySuccessfulResponseAndClaimIdFor(ClaimType.CARRIER);
+    super.testReadEobCarrierIdExpectClmIdInResponse();
   }
```

Comment:
I'd prefer to see these removed if they are not necessary.  They add some confusion and extra maintenance burden and another thing a maintainer would think was necessary to do when adding another test to the base class.

Reply:
Yeah I didn't love it. It seems using the Structure tab you mention we can run individual inherited tests without issue, so I'm ok to annotate in the base class and reduce the duplication. I did add some light instructions on how to run the individual tests in-IDE in the class doc, since it wasn't clear to me at the time.